kitsune/users/auth.py

```diff
@@ -90,6 +96,10 @@ def create_user(self, claims):
             add_to_contributors(user, profile.locale)
             del self.request.session["is_contributor"]
 
+        # If there is a refresh token, store it
+        if self.refresh_token:
```

Comment:
We either need to define this in`__init__` or use `getattr` to avoid the tests failing